<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/bottle_2_coke.xacro"/>
    <arg name="model_name" value="orange_juice"/>
    <arg name="z" value="0.75"/>
    <arg name="x" value="2.35"/>
    <arg name="y" value="-0.5"/>
  </include>
  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/bottle_2_fanta.xacro"/>
    <arg name="model_name" value="grenadine"/>
    <arg name="z" value="0.75"/>
    <arg name="x" value="2.35"/>
    <arg name="y" value="0.5"/>
  </include>
  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/bottle_2_pepsi.xacro"/>
    <arg name="model_name" value="tequila"/>
    <arg name="z" value="0.75"/>
    <arg name="x" value="2.35"/>
    <arg name="y" value="-0.9"/>
  </include>

  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/bottle_coke.xacro"/>
    <arg name="model_name" value="rum"/>
    <arg name="z" value="0.75"/>
    <arg name="x" value="2.35"/>
    <arg name="y" value="0.8"/>
  </include>
  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/bottle_fanta.xacro"/>
    <arg name="model_name" value="coke"/>
    <arg name="z" value="0.75"/>
    <arg name="x" value="2.35"/>
    <arg name="y" value="-1.2"/>
  </include>
  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/bottle_pepsi.xacro"/>
    <arg name="model_name" value="lime"/>
    <arg name="z" value="0.75"/>
    <arg name="x" value="2.35"/>
    <arg name="y" value="1.1"/>
  </include>

  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/yellow_glass.xacro"/>
    <arg name="model_name" value="glass_1"/>
    <arg name="z" value="0.9"/>
    <arg name="x" value="0.0"/>
    <arg name="y" value="0.0"/>
  </include>
  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/yellow_glass.xacro"/>
    <arg name="model_name" value="glass_2"/>
    <arg name="z" value="0.9"/>
    <arg name="x" value="0.0"/>
    <arg name="y" value="-1.0"/>
  </include>
  <include file="$(find tiago_bartender_world)/launch/spawn_xacro.launch">
    <arg name="xacro_file" value="$(find tiago_bartender_world)/robots/yellow_glass.xacro"/>
    <arg name="model_name" value="glass_3"/>
    <arg name="z" value="0.9"/>
    <arg name="x" value="0.0"/>
    <arg name="y" value="1.0"/>
  </include>
</launch>
